web/src/add_subscribers_pill.ts

```diff
@@ -132,6 +134,21 @@ export function create_without_add_button({
     return pill_widget;
 }
 
+export function append_user_group_from_name(
+    user_group_name: string,
+    pill_widget: CombinedPillContainer,
+): void {
+    const user_group = user_groups.get_user_group_from_name(user_group_name);
+    if (user_group === undefined) {
+        // This shouldn't happen, but we'll give a warning for now if it
+        // does.
+        blueslip.warn("User group with the given name does not exist.");
```

Comment:
This is one thing I'm not too sure about how to handle. Should I use `warn` or `error` here?

Reply:
error seems appropriate for a thing we think is impossible, since that will log to Sentry.

Reply:
Fixed [here](https://github.com/zulip/zulip/compare/e009972cafbcd6f5ca3f3a6ec5ff7f9111f0b5bc..2da7989c19501ca1751ae6e8a34ef7fe3aa97992)